bitnami/kubeapps/Chart.yaml

```
@@ -11,6 +11,7 @@ dependencies:
   - name: postgresql
     repository: https://charts.bitnami.com/bitnami
     version: 10.x.x
```

Please note the bitnami/postgresql version was updated in a major at https://github.com/bitnami/charts/pull/8827. Since this PR is also related to PostgreSQL, can you adapt this PR in order to use the latest version? 
```suggestion
    version: 11.x.x
```

---

Hmm, I'd be keen to avoid a major update of a dependency if it also means updating our own chart's major version as part of this change? Do you mind if we do that with our next release? I can update it in our dev chart already so it'll get synced when we release.

---

No problem, I already created an internal task to bump PostgreSQL and Redis versions in a major in a separate PR, but unfortunately, it is something that can't wait until the next release of the app itself since our internal test & release pipeline need to use the latest version of the dependencies to be protected against a possible security issue affecting one of these assets